CHANGES.md

```diff
@@ -5,6 +5,9 @@ If you're a contributor, please include your CHANGES entry in a file
 `doc/changes/$PR_NAME.md`. At release time, it will be incoporated into the
 changelog properly.
 
+- `dune utop` no longer links `utop` in "custom" mode, which should make this
```

Comment:
notice the comment above this message :)

Reply:
Argh, will fix. Thanks.